internal/install/static_snapshot_yml.go

```diff
@@ -102,14 +102,15 @@ services:
       fleet-server:
         condition: service_healthy
     healthcheck:
-      test: "sh -c 'grep \"Agent is starting\" -r . --include=elastic-agent-json.log'"
+      test: "elastic-agent status"
       retries: 90
       interval: 1s
     hostname: docker-fleet-agent
     environment:
     - "FLEET_ENROLL=1"
     - "FLEET_INSECURE=1"
     - "FLEET_URL=http://fleet-server:8220"
+    - "STATE_PATH=/usr/share/elastic-agent"
```

Comment:
Should we change it for both fleet-server and elastic-agent?

Reply:
I can try and change it, but previously it didn't work well (maybe it was the STATE_PATH issue). I will re-check it :)

Reply:
I updated the fleet-server to use "elastic-package status". Let's see.

Reply:
Why are you setting the `STATE_PATH`? The `/usr/share/elastic-agent/state` is the default, why not use that? Is this a workaround for the `status` command not working.

Reply:
@blakerouse Yes, otherwise the status command does not work as it does it looks into the wrong path.